static/css/documentation.css

```diff
@@ -1475,6 +1475,170 @@ pre span.comment {color: #aaa;}
 
 /* @end */
 
+/*------------------------------------------------------------------------------
+    Millennium Overview Table
+------------------------------------------------------------------------------*/
+#millennium-table-div {
+  width: 980px;
+  margin-top: 40px;
+  display: none;
+}
+
+#millennium-diff-table {
+  border-collapse: separate;
+  margin-left: -13px;
+}
+
+#millennium-diff-table .resource-name {
+  width: 150px;
+  text-align: left;
+}
+
+#millennium-diff-table .icon {
+  width: 16px;
+  padding: 6px 5px 6px 5px;
+}
+
+#millennium-diff-table thead tr th {
+  border: 0px;
+}
+
+#millennium-diff-table tr {
+  background-color: transparent;  
+}
+
+#millennium-diff-table td {
+  border: 0px;
+  text-align: center;
+  vertical-align: middle;
+}
+
+#millennium-diff-table tbody:nth-child(odd) {
+  background: #f9f9f9;
+}
+
+#millennium-diff-table tbody:hover td[rowspan], 
+#millennium-diff-table tr:hover td {
+   background: #f2f2f2; 
+}
+
+#table-key {
+  margin-top: 30px;
+}
+
+#table-failure-message {
+  padding: 15px;
+  border: 1px solid transparent;
+  border-radius: 4px;
+  color: #8a6d3b;
+  background-color: #fcf8e3;
+  border-color: #faebcc;
+  display: none;
+}
+
+#loading-spinner {
+  display: block;
+  position: relative;
+  width: 40px;
+  height: 40px;
+  margin: 0 auto;
+}
+
+#loading-spinner div {
+  box-sizing: border-box;
+  display: block;
+  position: absolute;
+  width: 30px;
+  height: 30px;
+  margin: 8px;
+  border: 4px solid #393d41;
+  border-radius: 50%;
+  animation: loading-spinner 1.2s cubic-bezier(0.5, 0, 0.5, 1) infinite;
+  border-color: #393d41 transparent transparent transparent;
+}
+#loading-spinner div:nth-child(1) {
+  animation-delay: -0.45s;
+}
+#loading-spinner div:nth-child(2) {
+  animation-delay: -0.3s;
+}
+#loading-spinner div:nth-child(3) {
+  animation-delay: -0.15s;
+}
+@keyframes loading-spinner {
+  0% {
+    transform: rotate(0deg);
+  }
+  100% {
+    transform: rotate(360deg);
+  }
+}
+
+[note-tooltip] {
```

Comment:
The tooltip has been renamed so that it is a data attribute: https://github.com/cerner/fhir.cerner.com/pull/416/commits/b55bf574995c3dbe2710d6a4734c8a1b90cde443 https://github.com/cerner/fhir.cerner.com/pull/416/commits/de755cb18506fe0370836bfed9a96d57296d7080#diff-4066a5d03340f019ca6531e5bc567919R395